<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dooby" failures="1" tests="1" name=" Add Customer Test" time="0.879" errors="0" timestamp="2021-04-02T13:23:43 IST">
  <testcase classname="com.dooby.testCases.AddCustomerTest" name="addCustomerTest" time="0.424">
    <failure type="java.lang.AssertionError" message="Failing intentionally">
      <![CDATA[java.lang.AssertionError: Failing intentionally
at org.junit.Assert.fail(Assert.java:88)
at com.dooby.testCases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
</testsuite> <!--  Add Customer Test -->
